edgexfoundry/edgex-compose
# feat: add secure mqtt message bus

Signed-off-by: Jim Wang (Intel) <yutsung.jim.wang@intel.com>
Signed-off-by: intel <joshua.silverio@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions

make gen dev mqtt-bus and check for usernamepassword / secretname: message-bus in mqtt-broker container
